web/site/app.vue

```diff
@@ -14,6 +15,13 @@ useHead({
 })
 
 const appVersion = await getAppMetaInfo()
+
+const { data: helpDocs } = await useAsyncData('help-docs-fetch', () => {
```

Comment:
query help docs in app.vue and provide globally. Originally I had this in the SbcHelpContent component but the prerender didnt work inside the modal